src/test/html-bundler_test.ts

```diff
@@ -193,8 +209,9 @@ suite('import-utils', () => {
         </dom-module>
         <script>Polymer({is: "my-element"})</script>`;
 
-      const ast = astUtils.parse(htmlBase);
-      importUtils.rewriteAstToEmulateBaseTag(analyzer, ast, 'the/doc/url');
+      const ast = parse(htmlBase);
+      htmlBundler['_rewriteAstToEmulateBaseTag'](
```

Comment:
Why does this use `['_rewriteAstToEmulateBaseTag']` style now?

Reply:
oh because it's a private method.

Reply:
the original tests were testing behavior of exported functions in the import-utils module but now they're all methods on the HtmlBundler and only a few methods need to be public on that thing because of the way it will be used.  To preserve the tests without testing for new behavior, I just call into the private methods directly.